static/js/markdown.js

```diff
@@ -564,3 +564,9 @@ export function apply_markdown(message) {
     message.flags = flags;
     message.is_me_message = is_status_message(raw_content);
 }
+
+export function parse_non_message(raw_content) {
+    // Occasionally we get markdown from the server that is not technically
+    // a message, but we want to convert it to HTML.
+    return marked(raw_content).trim();
+}
```

Comment:
Hmm, I wonder if we should be using the `helper_config` even when parsing non-messages; I'm pretty sure we do that for stream descriptions and the like.

Reply:
Haha, and of course the enxt commit does precisely that :).

Reply:
We should check that we're calling `clean_user_content_links` on any Markdown process output; that's a security measure we take in most places, but I think probably are not doing in the `zcommand` code path.

Reply:
Haha we are on the same page.  There is a super long commit message in the next commit that explains my thoughts.